README.md

```diff
@@ -46,15 +46,15 @@ A container is specified at `docker/Dockerfile`. To use it, execute the followin
 01. Build the image:
 
     ```bash
-    docker build . -f docker/Dockerfile -t cesarbr/knot-cloud-storage
+    docker build . -f docker/Dockerfile -t cesarbr/knot-cloud-storage-go
     ```
 
 01. Create a file containing the configuration as environment variables.
 
 01. Run the container:
 
     ```bash
-    docker run --env-file knot-cloud-storage.env -ti cesarbr/knot-cloud-storage
+    docker run --env-file knot-cloud-storage.env -ti cesarbr/knot-cloud-storage-go
```

Comment:
I think it's not necessary to change the image name for production because we will only use them when the migration is complete.

Reply:
You're right! Done!